tool2schema/parameter_schema.py

```diff
@@ -56,17 +56,14 @@ def _add_enum(self, schema: dict):
 
     def _add_description(self, schema: dict):
         """
-        Add the description of this parameter, extracted from the function docstring, to the given schema.
+        Add the description of this parameter, extracted from the function docstring, to the given
```

Comment:
Was this supposed to take two lines?

Reply:
Yeah otherwise it exceeds the 100 character line limit.